client/src/styles/global.scss

```diff
@@ -68,8 +68,8 @@
   &::before {
     color: white;
     background-color: #00a91c;
-    content: '✓'
+    content: "✓";
```

Comment:
Think this is an indication that prettier isn't running correctly for my ide setup?

Reply:
Ah yeah, that appears to have snuck in - this was formatted automatically when I ran `gatsby develop`